### PR TITLE
Use permission standard.

### PR DESCRIPTION
**Pull request by clesaege**

### CODE REVIEW

File: contracts/Blacklist.sol [hunk 1]
```diff
@@ -6,18 +6,27 @@ import "./Ownable.sol";
 contract Blacklist is BlacklistInterface, Ownable {
 
     mapping (bytes32 => bool) blacklisted;
-
+    
+    /** @dev Add a node to the blacklist.
```
Comment (decanus) [inline contracts/Blacklist.sol]: There should be a new line where the first comment is on, for all other comments.

**Review by decanus — CHANGES_REQUESTED** [review]

File: contracts/BlacklistInterface.sol [hunk 2]
```diff
@@ -4,9 +4,20 @@ interface BlacklistInterface {
...
+    /** @dev Remove a node from the blacklist.
+     *  @param node The node to remove from the blacklist.
+     */
     function unblacklist(bytes32 node) public;
-    function isBlacklisted(bytes32 node) public view returns (bool);
+    
+    /** @dev Return true if the node is allowed, false otherwise. Every nodes, except the blacklisted ones, are allowed.
+     *  @param node The node.
+     */
+    function isAllowed(bytes32 node) public view returns (bool);
```
Comment (decanus) [inline contracts/BlacklistInterface.sol]: Let's cut out the descriptions from here, along with the param declaration and only have them in ```Blacklist.sol```

**Review by decanus — CHANGES_REQUESTED** [review]

**Review by decanus — APPROVED** [review]

**Comment by decanus** [comment]
@clesaege you can merge this

**Comment by clesaege** [comment]
@decanus I can't, I don't have write access.

**Comment by decanus** [comment]
@clesaege, my bad. I'll merge

**Pull request merged**